### DISCUSSION TITLE
Welcome to Galaxy-Renderer Discussions!

### DISCUSSION THREAD

**Discussion by beltoforion** [beltoforion/Galaxy-Renderer#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clort81** [comment]
Hi! I found this with qwen3 "deep research" mode!
This is beautiful!
It compiles fine on devuan daedalus linux with amdgpu. Just 'make'.  No weirdness and NO PROBLEM!
WONDERFUL.

However amd laptop gpu is not enough for it.  Perhaps a very small galaxy could be made for default / low-end devices?
OpenGL Version Information:4.6 (Compatibility Profile) Mesa 22.3.6
- OpenGL:     4.6 (Compatibility Profile) Mesa 22.3.6
- GLSL:       4.60
- Vendor/GPU: AMD/AMD Radeon Vega 8 Graphics (raven, LLVM 15.0.6, DRM 3.61, 6.12.43+deb12-amd64)

**Comment by beltoforion** [comment]
Ultimately a simulation like this will need hardware acceleration. If you can compile it you can still reduce the number of stars in the 
third parameter of the call to wndMain.Init(...) here: https://github.com/beltoforion/Galaxy-Renderer/blob/master/main.cpp. 

And then there is still the typescript version: 
https://beltoforion.de/en/spiral_galaxy_renderer/spiral-galaxy-renderer.html
